tensorflow/tools/compatibility/tf_upgrade_test.py

```diff
@@ -134,3 +146,31 @@ def testInplace(self):
 
 if __name__ == "__main__":
   test_lib.main()
+
+"""tf.concat(0, [tf.concat(0, tiles[y]) for y in range(4)])
```

Comment:
Do you plan to use this comment later on? I am just making sure it is not a typo that you planned to remove.